packages/contracts/src/interfaces/IStakeManager.sol

```diff
@@ -179,6 +240,17 @@ interface IStakeManager {
      */
     function getBurnAddress() external view returns (address);
 
+    /**
+     * @notice Change the address that will receive the 'abandoned' stake.
```

Comment:
ok. the doc about "abondoned" relays is on "devAddress".
maybe add a 
struct AbndonedRelay {
  devAddress
  chewingDelay
  abandonedDelay
}

(for one thing, it adds a single method to retrieve those otherwise unused params. for another, it shows the relation between them...)

Reply:
In general I would be opposed to creating structs of three fields just to show they are related. It is generally much easier to work with basic types than structures in solidity and Web3.js, but if you really think it will be better I will join them into a struct.

Reply:
a struct (especial of "storage") is just a scope of several items (an alternative to a `structname_FieldName` uglifying)
It also has the benefit of having single method to set, and single method to retrieve.
btw: I think "OrphanRelayer" is a better name than "Abandoned", but its just a suggestion.

Reply:
Created a struct. 
Orphan is not a good name and does not convey what the intention is.